docs/src/security.md

```diff
@@ -293,28 +293,39 @@ CloudNativePG.
 : The instance manager requires to `update` and `patch` the status of any
   `Backup` resource in the namespace
 
-### Pod Security Policies
+### Security Contexts
 
-!!! Important
-    Starting from Kubernetes v1.21, the use of `PodSecurityPolicy` has been
-    deprecated, and as of Kubernetes v1.25, it has been completely removed. Despite
-    this deprecation, we acknowledge that the operator is currently undergoing
-    testing in older and unsupported versions of Kubernetes. Therefore, this
-    section is retained for those specific scenarios.
-
-A [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
-is the Kubernetes way to define security rules and specifications that a pod needs to meet
-to run in a cluster.
-For InfoSec reasons, every Kubernetes platform should implement them.
+A [Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+defines privilege and access control settings for a Pod or Container.
 
-CloudNativePG does not require *privileged* mode for containers execution.
-The PostgreSQL containers run as `postgres` system user. No component whatsoever requires running as `root`.
+CloudNativePG does not require *privileged* mode for container execution.
+The PostgreSQL containers run as the `postgres` system user. No component whatsoever requires running as `root`.
 
-Likewise, Volumes access does not require *privileges* mode or `root` privileges either.
+Likewise, Volume access does not require *privileges* mode nor `root` privileges.
 Proper permissions must be properly assigned by the Kubernetes platform and/or administrators.
 The PostgreSQL containers run with a read-only root filesystem (i.e. no writable layer).
 
-The operator explicitly sets the required security contexts.
+The operator explicitly sets the required security contexts for all pods and
+containers in the cluster. However, the
```

Comment:
The "However" here ... not sure. Configurability of the seccompProfile is not "against" the setting of the security context.